app/views/recipes/_add_to_meal_plan_dropdown.html.erb

```diff
@@ -1,18 +1,16 @@
 <%= form_for(meal_plan_recipe) do |form| %>
-  <% if available_meal_plans_dropdown(current_user, recipe).any? %>
```

Comment:
Unfortunately, i had to pull the "should i display this dropdown?" logic out of the partial and put it in 2 places. I wonder if there is a tidier way to do this. @mikevallano any ideas?

Reply:
🤔 Hmm. I'm not sure off the top. I poked around a bit, but it'd be easier you talked me through it so I have a better idea what's going on.

Reply:
The dropdown partial is just this dropdown and submit button
<img width="292" alt="Screenshot 2020-02-02 19 51 52" src="https://user-images.githubusercontent.com/8680712/73618410-8c37e980-45f5-11ea-9633-38ac824ef3c0.png">

It is used on the recipe#show page with a label that says "Add to Upcoming Plans" like this:
<img width="343" alt="Screenshot 2020-02-02 19 53 29" src="https://user-images.githubusercontent.com/8680712/73618448-d15c1b80-45f5-11ea-8cbc-344bf2f01497.png">

And also on the recipes#index here (without the label), like this: 
<img width="1124" alt="Screenshot 2020-02-02 19 53 17" src="https://user-images.githubusercontent.com/8680712/73618438-c43f2c80-45f5-11ea-81fc-854ed6ed130d.png">

The dropdown only appears if there are any `MealPlan`s with `start_date` in the future. I had that condition in the partial, but since i wanted a label in one place and not the other, i felt like i had to move the conditional too. It feels a little messy -- like i want to use a helper method to handle that condition, but i'm not really sure how to do it. 🤷‍♂